canonical/charmhub.io
# Fix markdown rendering on interface page

## Done
Fixed up the interface page markup and added mermaid to render charts from markdown

## How to QA
- Go to https://charmhub-io-1449.demos.haus/interfaces/ingress-v0
- Check that the markdown is rendered and that there is a flow chart diagram

## Issue / Card
Fixes https://warthogs.atlassian.net/browse/WD-665
